jsconfig.json

```diff
@@ -0,0 +1,5 @@
+{
+  "compilerOptions": {
```

Comment:
I've read `jsconfig.json` is a config file for VSCode, would you mind putting it in .gitignore, so it doesn't interfere with other people's IDEs? That way it should still work for you.